docs/advanced-topics/negative-mining.md

```diff
@@ -0,0 +1,91 @@
+(negative-mining)=
+# {octicon}`telescope` Negative Mining
+
+Negative Mining is an advanced technique, which optimizes the way data is sampled from you training dataset.
+Usually, it aims at making the metric learning tasks for the model harder during the training. 
```

Comment:
```suggestion
It works by slowing down metric learning, 
```

Reply:
I would rather say it speeds it up :laughing: but I think both does not really make sense.